PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/action/WhatsNewAction.java

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ */
+
+package com.microsoft.azure.toolkit.intellij.common.action;
+
+import com.intellij.ide.BrowserUtil;
+import com.intellij.ide.util.PropertiesComponent;
+import com.intellij.openapi.actionSystem.AnAction;
+import com.intellij.openapi.actionSystem.AnActionEvent;
+import com.intellij.openapi.fileEditor.FileEditor;
+import com.intellij.openapi.fileEditor.FileEditorManager;
+import com.intellij.openapi.fileTypes.FileType;
+import com.intellij.openapi.fileTypes.ex.FileTypeManagerEx;
+import com.intellij.openapi.project.Project;
+import com.intellij.openapi.util.Key;
+import com.intellij.openapi.util.io.StreamUtil;
+import com.intellij.openapi.vfs.VirtualFile;
+import com.intellij.testFramework.LightVirtualFile;
+import com.microsoft.azure.toolkit.intellij.common.messager.IntellijOpenInBrowserMessageAction;
+import com.microsoft.azure.toolkit.lib.common.exception.AzureToolkitRuntimeException;
+import com.microsoft.azure.toolkit.lib.common.messager.AzureMessager;
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import org.apache.commons.lang.StringUtils;
+import org.apache.maven.artifact.versioning.DefaultArtifactVersion;
+import org.intellij.plugins.markdown.ui.split.SplitFileEditor;
+import org.jetbrains.annotations.NotNull;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class WhatsNewAction extends AnAction {
+    private static final String WHATSNEW_URL = "https://github.com/microsoft/azure-tools-for-java/tree/develop/PluginsAndFeatures/" +
+            "azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/resources/whatsnew.md";
+    private static final String AZURE_TOOLKIT_FOR_JAVA = "What's New in Azure Toolkit for Java";
+    private static final String AZURE_TOOLKIT_WHATS_NEW = "AzureToolkit.WhatsNew";
+    private static final String VERSION_PATTERN = "<!-- Version: (.*) -->";
+    private static final String WHATS_NEW_CONTENT = "WHAT_S_NEW";
```

Comment:
-> WHATS_NEW_CONSTANT or WHATS_NEW_KEY? As this parameter only indicate the file type not the document body